docs/UserGuide.md

```diff
@@ -649,21 +649,26 @@ Adds a session to the session list.
 addses s/SESSION_NAME d/SESSION_DATE
 ```
 
-**Expected result:**
-```
-New session added: Tutorial 4 @ 12/07/2020
-```
+<div markdown="block" class="alert alert-info">
 
 **:information_source: Notes about the usage of this command:**<br>
 * Adds a session with the specified name and date to the displayed session list.
 * `SESSION_DATE` should be given in the format of d/M/yyyy or dd/MM/yyyy. This means if your day or month is single digit, you do not
 need to add a '0' in front, for e.g., 09/01/2020 and 9/1/2020 are both accepted.
 
-:bulb: **Tip:** Two sessions cannot have the same name but they can have the same date
+</div>
 
-Examples:
+:alert-warning: You will be exited from any currently entered session upon using the command.
+
+:green_book: Examples:
 * `addses s/tut1 d/12/7/2020`
 
+:bulb: **Tip:** Two sessions cannot have the same name but they can have the same date
```

Comment:
Perhaps these parts could be wrapped in their respective div boxes for better organisation of info?